src/tree_sitter/parser.h

```diff
@@ -129,16 +130,9 @@ struct TSLanguage {
  *  Lexer Macros
  */
 
-#ifdef _MSC_VER
-#define UNUSED __pragma(warning(suppress : 4101))
-#else
-#define UNUSED __attribute__((unused))
-#endif
-
 #define START_LEXER()           \
   bool result = false;          \
   bool skip = false;            \
-  UNUSED                        \
```

Comment:
Could you explain what happened here? Otherwise, LGTM

Reply:
I honestly have no idea. This header is auto-generated, to the best of my knowledge, by `tree-sitter generate`. My hypothesis is that the last merged PR (#29), which was authored by someone else, used a different version of `tree-sitter` than I do. I'm using the latest one available from Nixpkgs (20.x), but the last release is 21.x. Because it's not reasonable to ask all contributors to use the latest one - especially if not available in a repo like Nixpkgs yet - I think it's ok as long as contributors use a version reasonably enough. What do you think?